src/transformers/utils/quantization_config.py

```diff
@@ -418,3 +429,61 @@ def post_init(self):
                     f"""dataset needs to be either a list of string or a value in
                     ['wikitext2','c4','c4-new','ptb','ptb-new'], but we found {self.dataset}"""
                 )
+
+
+@dataclass
+class AWQConfig(QuantizationConfigMixin):
+    """
+    This is a wrapper class about all possible attributes and features that you can play with a model that has been
+    loaded using `auto-awq` library awq quantization relying on auto_awq backend.
+
+    Args:
+        w_bit (`int`):
+            The number of bits to quantize to.
+        q_group_size (`int`, *optional*, defaults to 128):
+            The group size to use for quantization. Recommended value is 128 and -1 uses per-column quantization.
+        zero_point (`bool`, *optional*, defaults to `True`):
```

Comment:
I changed the attribute names 
w_bit --> bits / `q_group_size` -> `group_size` to be consistent with GPTQ. I don't think we should care about BC here since the PR has not been merged yet